android/src/main/kotlin/live/hms/hmssdk_flutter/views/HMSVideoView.kt

```diff
@@ -25,12 +33,89 @@ class HMSVideoView(context: Context,setMirror:Boolean) : ConstraintLayout(contex
 
         surfaceViewRenderer = view.findViewById(R.id.surfaceViewRenderer)
         surfaceViewRenderer.setEnableHardwareScaler(true)
-        Log.i("HMSVideoViewAndroid",setMirror.toString())
+        Log.i("HMSVideoViewAndroid", setMirror.toString())
         surfaceViewRenderer.setMirror(setMirror)
-        surfaceViewRenderer.setScalingType(RendererCommon.ScalingType.SCALE_ASPECT_FILL)
+        surfaceViewRenderer.setScalingType(RendererCommon.ScalingType.SCALE_ASPECT_FIT)
```

Comment:
should come from clients, default to `SCALE_ASPECT_FIT`

Reply:
resolved